spark-print-data/printertypes.json

```diff
@@ -656,10 +656,100 @@
                 "z_axis_feed_rate": 3
             },
             "_files": [
-				"icons.16x16_id",
-				"icons.32x32_id",
-				"icons.50x50_id",
-				"icons.100x100_id",
+                "icons.16x16_id",
+                "icons.32x32_id",
+                "icons.50x50_id",
+                "icons.100x100_id",
+                "build_volume.bed_file_id"
+            ]
+        }, 
+        {
+            "id" : "D4H4F6G4-8J4H-5F6K-4F3J-4GK5FH4H56GF",
+            "version" : 1,
+            "name" : "DreamMaker_OverLord",
+            "manufacturer" : "DFRobot",
+            "registration_url" : null,
+            "model_number" : "1.0.0",
+            "icons" : {
+                "16x16_id" : "data/DreamMaker_OverLord_16x16.png",
```

Comment:
you have the underscore here in the filename, but the assets you have added do not have the "_" so they cannot be found